### DISCUSSION TITLE
rclone: too many non-flag arguments

### DISCUSSION THREAD

**Discussion by Amphiboly** [ep1cman/unifi-protect-backup#21] [Q&A] [answered]
It appears that poorly formatted file names are being passed to rclone. I get the message:
Command rcat needs 1 arguments maximum: you provided 2 non flag arguments: ["'remote:Unifi\\Wanderer\\2022-03-07\\2022-03-07T03-20-06" "motion.mp4'"]
when my camera name has no spaces, and
Command rcat needs 1 arguments maximum: you provided 4 non flag arguments: ["'remote:Unifi\\Mud" "Room" "Door\\2022-03-07\\2022-03-07T03-20-05" "motion.mp4'"]
when it has two spaces. So even if I were to rename the camera "Front Door" to "FrontDoor" it looks like there will still be a problem because of the "motion.mp4" addition.

This is on Windows 11 with the current versions of all involved packages (Python 3.10.2, unifi-protect-backup 0.5.0, pyunifiprotect 3.2.1, ffprobe 0.5, rclone 1.57.0).

unifi-protect generates test data per the readme, and rclone rcat can put files into the target directory.

What should I look at next?

-- 
rik

**Comment by ep1cman** [comment]
Hi,

I've create an issue #22 from this, this is definitely a bug.

**Comment by ep1cman** [comment, marked as answer]
Should be resolved by v0.5.1
